packages/main/src/List.js

```diff
@@ -575,6 +575,7 @@ class List extends UI5Element {
 
 	initItemNavigation() {
 		this._itemNavigation = new ItemNavigation(this, {
+			skipItemsSize: 10, // PAGE_UP and PAGE_DOWN will skip trough 10 items
```

Comment:
@nnaydenow I know we talked about 10, but could you check the spec as in OpenUI5 I can see that 20 items are skipped.